api/src/main/resources/swagger3codegen/aouTracedMethodCall.mustache

```diff
@@ -0,0 +1,34 @@
+        // Generate an OpenCensus tracing span for this API method call.
```

Comment:
@gjuggler FYI, I have spawned an additional set of templates here to support swagger 3. The patch was relatively straightforward to apply against the swagger3 codegen - these templates didn't change much.